src/app/helpers/sendEmail.ts

```diff
@@ -0,0 +1,23 @@
+import nodemailer = require("nodemailer")
+import { CoachInterface } from "../../interfaces/CoachInterface"
+export default (aCoach: CoachInterface) => {
+
+    let transporter = nodemailer.createTransport({
+        service: "gmail",
+        auth: {
+            user: 'coach.red.proto', // gmail user
+            pass: 'Mayden123', // gmail password
```

Comment:
If these are real creds id consider putting them into a config.json files that gets imported: (The second answer) https://stackoverflow.com/questions/5869216/how-to-store-node-js-deployment-settings-configuration-files